less/global.less

```
@@ -14,33 +14,49 @@ h1, h2, h3, h4, h5 {
   font-family: 'Indie Flower', cursive;
 }
 
+
 h1 {
   font-size: 4rem;
+  @media(max-width: 500px) {
+    font-size: 2.3rem;
+  }
 }
 
 h2 {
   font-size: 3.2rem;
   padding-bottom: 10px;
+  @media(max-width: 500px) {
```

I would have liked to see the use of interpolation here.
```suggestion
  @media @phone {
```